updatecli/values.yaml

```diff
@@ -0,0 +1,5 @@
+github:
+  user: "UpdateCLI"
+  email: "updatecli@opzkit.io"
+  username: "github-actions"
+  token: "UPDATECLI_GITHUB_TOKEN"
```

Comment:
```suggestion
```

This line could be removed as you use 

```
      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
```

instead of 

```
      token: '{{ requiredEnv .github.token }}'
```

The end result doesn't change much but your approach is more readable